packages/doge/build.sh

```diff
@@ -4,8 +4,9 @@ TERMUX_PKG_LICENSE="EUPL-1.2"
 TERMUX_PKG_LICENSE_FILE="LICENCE"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=0.2.8
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/Dj-Codeman/doge/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
```

Comment:
I think we should also update the SRCURL to point at the new name instead of relying on the redirect then.
```suggestion
TERMUX_PKG_SRCURL=https://github.com/Dj-Codeman/dog_community/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
```

Reply:
Sure, that also works, I had just left it that way since the entire project wouldn't be technically renamed until the next release. At that point, the binary name might change or might not, and then it might or might not make sense to rename the whole package again.